scanomatic/ui_server/status_api.py

```diff
@@ -52,7 +52,7 @@ def status_scanners(status_query=None):
                 scanners={
                     s['socket']: s['scanner_name']
                     for s in rpc_client.get_scanner_status()
-                    if 'owner' not in s or not s['owner']
+                    if s.get('owner', {}).get('pid', 0) == 0
```

Comment:
what is the logic for this here, it looks odd

Reply:
The real problem is that the "new model" initiates a free scanner object with  {'owner': 'id':0}  but the old parts of scanomatic considers an object with out 'owner' as a free scanner.

Reply:
does that mean that the `pid` no longer is optional / can that be updated on the model typing?